physionet-django/user/management/commands/utility.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from django.utils import timezone
+
+from user.models import CredentialApplication
+
+
+def get_pending__reference_response_credentialing_application(n, days, ignore_reminded_application=False):
+    """
+    Get n CredentialApplication that has been pending reference response for more than the given days.
+    """
+    today = timezone.now()
+    limit = today - timezone.timedelta(days=days)
+
+    # get all applications that have decision pending
+    applications = CredentialApplication.objects.filter(status=0)
+    # get applications with Reference Response pending
+    response_applications = applications.filter(credential_review__status=30)
+    # ignore applications that have been reminded already
+    if ignore_reminded_application:
```

Comment:
Maybe the default should be set to True?

Reply:
the idea was,

- to add the parameter in a way that we wont need to tough the existing use of the function such as when autorejecting applicants. For that case we dont want to ignore the reminded application(we want to get all applications that are past the configured days)

So currently in `remind_pending_apps` we explicitly do `ignore_reminded_application=True` and make sure we dont reset the notification to the same person


may i ask why you like the default set to True?

Reply:
Thanks Amit, you have a good point. Let's leave the default as False. I'd suggested True because I felt that we would always want to avoid re-chasing people who had already been chased, but on reflection I think it is better to leave as False for the reason you suggest (i.e. more predictable behaviour).

Reply:
Thank you. just saw some typo on my comment and fixed that.